src/backend/InvenTree/stock/models.py

```diff
@@ -458,44 +459,98 @@ def report_context(self):
         & Q(expiry_date__lt=InvenTree.helpers.current_date())
     )
 
-    def update_serial_number(self):
-        """Update the 'serial_int' field, to be an integer representation of the serial number.
+    @classmethod
+    def create_serial_numbers(cls, serials: list, **kwargs) -> QuerySet:
```

Comment:
C: I think this should be an internal method (_ prefix) as it should not be used by plugins etc.

Reply:
Fair enough, I will do so.